authentik/core/models.py

```diff
@@ -274,6 +274,9 @@ class User(SerializerModel, GuardianUserMixin, AttributesMixin, AbstractUser):
     ak_groups = models.ManyToManyField("Group", related_name="users")
     password_change_date = models.DateTimeField(auto_now_add=True)
 
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
`AbtractUser` already has this a `date_joined`

Reply:
Ah nice! I'll remove then this field then.

Reply:
We have a `CreatedUpdatedModel` which uses `created` and `last_updated` as field names. Let's align on `last_updated`, and leave the other one as is to not introduce a breaking change in the API.